Javascript/q47.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>q47</title>
+
+  <style>
+    body {
+      font-family: Verdana, Geneva, Tahoma, sans-serif;
+    }
+
+    .flex {
+      display: flex;
+      gap: 10px;
+    }
+  </style>
+
+  <script>
+    var val = 0;
+    function myFunc() {
+    document.getElementById("demo0").innerHTML= val--;
```

Comment:
The question says "Minimum allowed value is 0". You do not have any protection against negative values